akka-cluster-sharding/build.sbt

```diff
@@ -0,0 +1,18 @@
+import akka.{ AkkaBuild, Dependencies, Formatting, MultiNode, Unidoc, OSGi }
+import com.typesafe.tools.mima.plugin.MimaKeys
+
+AkkaBuild.defaultSettings
+
+Formatting.formatSettings
+
+Unidoc.scaladocSettingsNoVerificationOfDiagrams
+
+Unidoc.javadocSettings
+
+OSGi.clusterSharding
+
+MultiNode.multiJvmSettings
+
+libraryDependencies ++= Dependencies.clusterSharding
```

Comment:
the libraries can be defined centrally, but the specific usage of them can be defined here